.editorconfig

```diff
@@ -1,6 +1,6 @@
 root = true
 
-[**.{cpp,h,java,mm,cs,ts,js,tsx}]
+[**.{java,mm,cs,ts,js,tsx}]
```

Comment:
need .m

Reply:
what's .m?

Reply:
sorry, not .m, i meant m

mm means objective-c++
m means objective-c

Reply:
You can create a separate PR. This PR only do removing on .editorconfig